.github/workflows/lint.yml

```diff
@@ -23,4 +23,5 @@ jobs:
           bundler-cache: false
       - uses: r7kamura/rubocop-problem-matchers-action@v1  # this shows the failures in the PR
       - run: |
+          bundle install --jobs 4 --retry 3
```

Comment:
huh - ruby-setup should do this, ya sure?

Reply:
My bad.. it was the Lint.yml file that fails with that error message, see below

Reply:
Oh. That's because you have `bundler_cache: true` for ruby-setup, which is useful if you have many jobs that are going to use that same cache. IN this case you can drop bundler_cache:true (or you can keep the manual install if you prefer).

Reply:
Line 23 says cache is false. Experimentation time

Reply:
Hmmm, maybe I'm wrong, maybe it just installs the right version of bundler. Carry on,

Reply:
OK, remove the cache line completely and it still failed. I fell back to "bundle install" and that seems to make it happy

Reply:
LGTM!